lib/SPIRV/SPIRVUtil.cpp

```diff
@@ -2032,7 +2032,7 @@ static void replaceUsesOfBuiltinVar(Value *V, const APInt &AccumulatedOffset,
           if (!Index.isZero() || DL.getTypeSizeInBits(VecTy) !=
                                      DL.getTypeSizeInBits(Load->getType()))
             llvm_unreachable("Illegal use of a SPIR-V builtin variable");
-          Replacement = UndefValue::get(VecTy);
+          Replacement = PoisonValue::get(VecTy);
```

Comment:
With this change you should probably also update the `%2 = insertelement <3 x i64> undef, i64 %1, i32 0` in `docs/SPIRVRepresentationInLLVM.rst`?